src/components/Accordion/Accordion.Heading.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import { HeadingUI } from './Accordion.css'
+
+export default ({ children }) => (
+  <HeadingUI truncate weight={500} size={14} className="Accordion__Heading">
```

Comment:
I am fine leaving it like that (in parts because I think we should target component as selector over classNames), but the convention I believe is to prefix className with `c-`